### DISCUSSION TITLE
Joint always is created on 0 x,y,z Origin

### DISCUSSION THREAD

**Discussion by amunioz** [AlfMikael/snap-generator#11] [Q&A]
Hi new here
Testing this cool tool but cant find thw way to create a joint on a custom joint origin point.
What can I doing wrong?
Thx

**Comment by AlfMikael** [comment]
Hey

The option to position at a custom joint origin became broken with the latest version of Fusion. It will be fixed soon.

**Comment by amunioz** [comment]
Oh!
Ok.
I'll wait for the fix
I think this tool is awesome
Thanks a lot for share your work.
Regards

**Reply by AlfMikael** [reply]
Should be fixed now :)
